upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -280,6 +281,10 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap, secretStore fi.SecretS
 		dest["EnableSQSTerminationDraining"] = func() bool { return *cluster.Spec.NodeTerminationHandler.EnableSQSTerminationDraining }
 	}
 
+	dest["SupportedArchitecture"] = tf.SupportedArchitecture
+
+	dest["ParseTaint"] = parseTaint
```

Comment:
Maybe `ExpandTaint` ?

Reply:
The function name came from copying the k/k function. I suggest we keep that link even though I may agree ExpandTaint is more descriptive.